tests/chainer_tests/dataset_tests/tabular_tests/test_slice.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+import unittest
+
+from chainer import testing
+from chainer.dataset import TabularDataset
```

Comment:
Do not import a class; import a module instead (openstack guideline). Ditto for line 7.